src/Microsoft.Health.Fhir.SqlServer/Features/Storage/TvpRowGeneration/V3/CompartmentAssignmentRowGenerator.cs

```diff
@@ -0,0 +1,97 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading;
+using EnsureThat;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.SqlServer.Features.Schema.Model;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Storage.TvpRowGeneration.V3
+{
+    internal class CompartmentAssignmentRowGenerator : ITableValuedParameterRowGenerator<ResourceMetadata, Schema.Model.V3.CompartmentAssignmentTableTypeRow>
```

Comment:
Hmm, I think we need to fix our codegen, since these table types are immutable and cannot change. So we should not need to have all this copy-paste. I'll open an issue.